iree/turbine/kernel/compiler/host_codegen.py

```diff
@@ -80,13 +80,22 @@ def isolated_test_call(
             dispatch = SymbolRefAttr.get([exe.sym_name.value, entrypoint])
             entrypoints = ArrayAttr.get([dispatch])
 
+            buffer_binding_count = len(sig.kernel_buffer_bindings)
+            offset_to_out_binding = len(sig.kernel_buffer_input_bindings)
+            tied_operands = ArrayAttr.get(
+                [
+                    IntegerAttr.get(IndexType.get(), out_idx)
+                    for out_idx in range(offset_to_out_binding, buffer_binding_count)
+                ]
+            )
             out = flow_d.DispatchOp(
                 output_tensors,
                 [dynamic_argument_map[dim] for dim in dynamic_symbols],
                 entrypoints,
                 entry_block.arguments,
                 [dynamic_argument_map[dim] for dim in argument_dims],
                 [dynamic_argument_map[dim] for dim in result_dims],
+                tied_operands=tied_operands,
```

Comment:
This is pretty cool. So if operand 2 is tied, does that mean that the output will be written to the memory allocated for operand 2 and so we won't have to allocate new memory?

Reply:
exactly! :)

Reply:
very nice :)